### PR TITLE
Update style aspects of manual-multiple template

### PR DESCRIPTION
**Pull request by RobertFreeman**
- Change button colour in header div and padding on blurb for manual-soulmates container

### CODE REVIEW

File: src/_shared/scss/_adverts.scss [hunk 1]
```diff
@@ -569,12 +569,16 @@
...
+        color: #000000;
     }
 
     .adverts__blurb {
         font: normal 14px/18px $f-serif-headline;
-        color: #ffffff;
+        margin-top: 20px;
+        color: #000000;
+        background: #ffffff;
+        -webkit-box-decoration-break: clone;
```
Comment (JonNorman) [inline src/_shared/scss/_adverts.scss]: Just use `box-decoration-break` here; the transpiler will add all of the necessary variants when it builds the templates.

File: src/_shared/scss/_core.scss [hunk 2]
```diff
@@ -112,7 +112,7 @@ a.u-faux-block-link__overlay {
 .button {
     border: 0;
     border-radius: 1000px;
-    font: bold 13px/20px $f-sans-serif-text;
+    font: bold 13px/0px $f-sans-serif-text;
```
Comment (JonNorman) [inline src/_shared/scss/_core.scss]: Does this not make the font-size 0?

File: src/_shared/scss/_adverts-books.scss [hunk 3]
```diff
@@ -1,3 +1,8 @@
+.adverts--tone-books .button--large {
```
Comment (JonNorman) [inline src/_shared/scss/_adverts-books.scss]: This seems to be present, unchanged, in a lot of files - is it worth factoring it out in a common file - maybe `_adverts.scss`?

Comment (JonNorman) [reply src/_shared/scss/_adverts-books.scss]: (It doesn't seem to depend on being books, jobs etc.)

File: src/_shared/scss/_adverts.scss (hunk 1 above)
Comment (RobertFreeman) [reply src/_shared/scss/_adverts.scss]: I'll change that - sorry, you told me that already...

File: src/_shared/scss/_core.scss (hunk 2 above)
Comment (RobertFreeman) [reply src/_shared/scss/_core.scss]: It makes the line height 0px - I did that to vertical-align the button text in the centre (the studio said the text was too high and mis-aligned with the arrow), sure there's a better way of doing that.

File: src/_shared/scss/_adverts-books.scss (hunk 3 above)
Comment (RobertFreeman) [reply src/_shared/scss/_adverts-books.scss]: Definitely that would be much cleaner - I'm going to try and refactor the code in the next couple of weeks, hopefully remove unnecessary files and code...

File: src/_shared/scss/_core.scss (hunk 2 above)
Comment (JonNorman) [reply src/_shared/scss/_core.scss]: Hmm, what is the content of the button? If there isn't any text then presumably we can just set `line-height: 0`; maybe we can look at this together and see if there is a cleaner way of achieving this (if `line-height` doesn't work)

Comment (RobertFreeman) [reply src/_shared/scss/_core.scss]: I just checked and we can just delete the line-height bit altogether I think and that'll do the same thing. The button small div is:

image

Comment (JonNorman) [reply src/_shared/scss/_core.scss]: Nice one, deleting code is always a bonus 😃

**Pull request closed without merging**